src/v2/api/controllers/UserController.ts

```diff
@@ -462,28 +474,85 @@ export class UserController {
   @ApiParam({
     name: 'userId',
     type: String,
+    description: 'Id of a user to delete contact',
   })
   @ApiParam({
     name: 'contactId',
     type: String,
+    description: 'Id of a contact to delete',
+  })
+  @ApiEndpoint({
+    summary: 'Request captain or admin to delete the contact',
+    permissions: PERMISSION.USERS_$USERID_CONTACTS_DELETE,
   })
-  @Access(PERMISSION.USERS_$USERID_CONTACTS_DELETE)
   @Delete('/:userId/contacts/:contactId')
   deleteContact (
-    @Param(ContactByUserIdPipe) params,
+    @Param(ContactByUserIdPipe) params: { userId: string, contactId: string },
   ) {
     return this.userService.deleteContact(params.userId, params.contactId);
   }
 
-  @Access(PERMISSION.STUDENTS_DELETE)
+  @ApiBearerAuth()
+  @ApiOkResponse()
+  @ApiUnauthorizedResponse({
+    description: `\n
+    UnauthorizedException:
+      Unauthorized`,
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidEntityIdException:
+      User with such id is not found`,
+  })
```

Comment:
add @ApiForbiddenResponse with "NoPermissionException: You do not have permission to perform this action"